From 9612d37dcadc1423c49de58f0b8e093ae3a61383 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sun, 2 Dec 2012 08:54:47 +0100
Subject: [PATCH 9/9] Fix build with GCC 4.6 -Werror=unused-but

* similar to c200e893ccd29c2d6250bcc08f5f6c245a504c1a but for s3c2410
  too

* This patch fixes the following build failures:
  src/cpu/s3c2410/s3c24xx-mci.c: In function 'mmc_block_read':
  src/cpu/s3c2410/s3c24xx-mci.c:140:7: error: variable 'resp' set but not used [-Werror=unused-but-set-variable]

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/cpu/s3c2410/s3c24xx-mci.c | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/src/cpu/s3c2410/s3c24xx-mci.c b/src/cpu/s3c2410/s3c24xx-mci.c
index 65a9513..8008995 100644
--- a/src/cpu/s3c2410/s3c24xx-mci.c
+++ b/src/cpu/s3c2410/s3c24xx-mci.c
@@ -137,7 +137,6 @@ static int mmc_block_read(u8 *dst, u32 src, u32 len)
 {
 	u32 dcon, fifo;
 	u32 *dst_u32 = (u32 *)dst;
-	u32 *resp;
 
 	if (len == 0)
 		return 0;
@@ -145,7 +144,7 @@ static int mmc_block_read(u8 *dst, u32 src, u32 len)
 //	debug("mmc_block_rd dst %lx src %lx len %d\n", (u32)dst, src, len);
 
 	/* set block len */
-	resp = mmc_cmd(MMC_CMD_SET_BLOCKLEN, len, CMD_F_RESP);
+	mmc_cmd(MMC_CMD_SET_BLOCKLEN, len, CMD_F_RESP);
 	SDIBSIZE = len;
 
 	//SDIPRE = 0xff;
@@ -164,9 +163,9 @@ static int mmc_block_read(u8 *dst, u32 src, u32 len)
 
 	/* send read command */
 	if (!is_sdhc)
-		resp = mmc_cmd(MMC_CMD_READ_BLOCK, src, CMD_F_RESP);
+		mmc_cmd(MMC_CMD_READ_BLOCK, src, CMD_F_RESP);
 	else
-		resp = mmc_cmd(MMC_CMD_READ_BLOCK, src / MMC_BLOCK_SIZE, CMD_F_RESP);
+		mmc_cmd(MMC_CMD_READ_BLOCK, src / MMC_BLOCK_SIZE, CMD_F_RESP);
 
 	while (len > 0) {
 		u32 sdidsta = SDIDSTA;
-- 
1.8.0

